### ISSUE TITLE
[Suggestion] Make evolution chamber automatable

### ISSUE THREAD

**Issue by Miradae** [Zairus/MegaLoot#18]
At the moment it's not possible to extract and import items from/into the evolution chamber via hoppers/cables etc (it's not even possible to shift click them). That makes it extremely annoying to upgrade a bunch of loot cases (at our server, many people trade them, so it's reasonable to get a few stacks, at least the common ones).
My suggestion would be to just enable the extraction/import capability of the evolution chamber via external means (be it hoppers or 3rd party cables).

**Comment by PurpleHel** [comment]
Yes, this, *please*! I spent half an hour of actual time standing there upgrading cases yesterday!
